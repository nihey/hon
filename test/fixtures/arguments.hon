# A long time ago, at a distance volcanic planet, on a galaxy far, far away.

python
    print "$1 - It's over $2, I have the high ground."

node
    console.log("$2 - You underestimate my power.");

python
    print "$1 - Don't try it."

echo "[$1 slices $2's legs with a light saber]"

python
    print "$1 - You were the chosen one."
    print "$1 - It was said that you would destroy the Sith, not join them."
    print "$1 - Bring ballance to the force, not leave it in darkness."

node
    console.log("$2 - I hate you!!!");

python
    print "$1 - You were my brother $2, I loved you."
