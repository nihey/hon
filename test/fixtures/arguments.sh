#!/bin/bash

echo "print \"$1 - It's over $2, I have the high ground.\"" | python
echo "console.log(\"$2 - You underestimate my power.\");" | node
echo "print \"$1 - Don't try it.\"" | python
echo "[$1 slices $2's legs with a light saber]"
echo "print \"$1 - You were the chosen one.\";print \"$1 - It was said that you would destroy the Sith, not join them.\";print \"$1 - Bring ballance to the force, not leave it in darkness.\"" | python
echo "console.log(\"$2 - I hate you!!!\");" | node
echo "print \"$1 - You were my brother $2, I loved you.\"" | python