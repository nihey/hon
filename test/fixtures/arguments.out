Obi-Wan - It's over Anakin, I have the high ground.
Anakin - You underestimate my power.
Obi-Wan - Don't try it.
[Obi-Wan slices Anakin's legs with a light saber]
Obi-Wan - You were the chosen one.
Obi-Wan - It was said that you would destroy the Sith, not join them.
Obi-Wan - Bring ballance to the force, not leave it in darkness.
Anakin - I hate you!!!
Obi-Wan - You were my brother Anakin, I loved you.
